Author: Francois Romieu <romieu@fr.zoreil.com>
Date: Fri, 27 May 2005 19:11:33 +0000 (+0200)
Source: http://www.kernel.org/git/gitweb.cgi?p=linux/kernel/git/torvalds/linux-2.6.git;a=commitdiff;h=a7b6459de19c85432f791728317089459316da32

  [PATCH] r8169: new PCI id
  
  The USR 997902 is based on the 8169 chipset.
  
  The value has been extracted from the sources of the driver which
  comes with the manufacturer's cdrom. Heads-up and test by TommyDrum
  <mycooc@yahoo.it>.
  
  Signed-off-by: Francois Romieu <romieu@fr.zoreil.com>

--- a/drivers/net/r8169.c
+++ b/drivers/net/r8169.c
@@ -176,6 +176,7 @@ const static struct {
 static struct pci_device_id rtl8169_pci_tbl[] = {
 	{0x10ec, 0x8169, PCI_ANY_ID, PCI_ANY_ID, 0, 0, 0},
 	{0x1186, 0x4300, PCI_ANY_ID, PCI_ANY_ID, 0, 0, 0},
+	{0x16ec, 0x0116, PCI_ANY_ID, PCI_ANY_ID, 0, 0, 0},
 	{0,},
 };
 
